Source/ASCollectionView.mm

```diff
@@ -64,8 +64,6 @@
     return __val; \
   }
 
-#define ASIndexPathForSection(section) [NSIndexPath indexPathForItem:0 inSection:section]
```

Comment:
Removed this macro because using item 0 is only something that's done for flow layout headers and footers. In any other context, "index path for section" would mean a one-node index path.